src/css/application.css

```diff
@@ -483,6 +483,10 @@ body {
   background-color: var(--color-low-contrast-gray);
 }
 
+.console__button_clear {
+  float: right;
```

Comment:
nooooooooo anything but `float`!

(practical suggestion: either flexbox or just absolutely position this)

Reply:
Hahaha sounds good! Saw it used in the preview button (pop out and refresh) so thought I'd give it a try. Will update!